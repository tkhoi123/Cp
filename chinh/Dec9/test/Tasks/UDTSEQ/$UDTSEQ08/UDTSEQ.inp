3 650
3 3 955
2 2 420
2 3 -830
2 3 344
3 3 -59
3 3 -117
1 1 -883
1 2 -818
1 2 940
2 3 -567
3 3 850
1 2 193
3 3 -569
3 3 650
2 2 401
1 3 -695
1 1 -763
2 3 -769
3 3 254
2 2 581
3 3 169
2 3 -663
2 2 -513
1 3 932
3 3 -426
2 3 -749
1 1 -471
2 2 -852
1 3 288
1 1 991
1 2 -411
2 2 -813
1 3 -557
2 3 -185
1 1 -755
2 3 746
2 2 -487
3 3 25
3 3 -751
2 3 -59
2 2 -752
1 2 -164
1 1 566
2 2 -340
1 1 150
1 3 697
1 2 -664
3 3 58
1 2 450
2 3 404
1 1 -558
1 1 -729
2 3 29
1 2 166
2 2 -323
1 3 -739
1 3 -292
1 2 -457
2 2 -615
1 3 -701
3 3 -927
1 2 -128
1 1 609
1 1 -451
3 3 -197
1 1 -388
2 3 649
1 2 514
1 1 -167
1 1 37
2 3 383
3 3 529
1 1 720
2 3 -275
3 3 -691
2 3 124
2 3 -220
3 3 -510
1 1 804
1 3 514
1 3 681
1 2 -273
1 2 256
1 1 -159
1 2 -700
2 2 270
1 1 -923
2 3 838
1 3 200
1 3 395
3 3 -630
1 3 -325
1 1 149
2 2 18
2 2 -512
2 3 -900
1 1 -450
2 3 -774
1 3 926
1 1 469
3 3 550
2 3 -703
1 3 805
1 3 96
1 1 494
1 1 -519
2 2 171
2 2 268
1 1 839
1 1 178
3 3 -519
2 3 592
2 2 -276
2 3 -359
2 3 249
2 3 -618
2 2 -606
1 3 580
3 3 -15
3 3 -174
2 2 -812
2 3 795
2 2 594
2 2 705
3 3 245
2 2 763
1 3 206
1 2 291
2 2 515
1 2 218
3 3 794
1 1 -88
1 3 479
2 3 -470
2 3 695
3 3 -704
1 1 -518
2 3 218
2 2 -669
1 3 583
3 3 946
2 2 -701
1 1 949
2 3 236
1 2 739
1 1 -161
1 2 -562
2 2 257
2 2 959
1 3 431
1 1 322
1 3 57
3 3 981
2 2 20
3 3 -759
1 2 -540
2 3 261
1 3 -985
3 3 37
2 2 -234
2 3 1
1 2 -502
2 3 465
2 2 -902
1 3 -69
2 3 -90
3 3 88
1 1 398
1 1 272
3 3 -694
2 3 320
3 3 -506
3 3 492
1 3 697
2 2 49
2 3 -645
1 3 -214
1 1 603
2 3 -85
1 3 42
2 2 -249
1 3 222
1 3 -833
1 1 -39
2 2 14
1 2 8
2 2 -6
1 2 -162
1 1 712
3 3 273
2 3 -410
2 2 956
3 3 -443
2 2 -349
2 3 28
2 3 266
1 2 -813
1 2 -501
2 3 444
2 2 -143
1 1 -856
2 2 -183
1 1 -700
3 3 -351
1 1 -602
1 1 -898
2 3 60
2 2 -653
3 3 618
1 3 259
2 2 -535
2 3 900
2 2 514
2 3 -835
2 3 -827
3 3 392
1 2 986
1 2 -63
2 2 126
3 3 -695
1 2 298
1 1 119
2 2 887
3 3 415
2 2 63
1 2 852
1 1 625
2 3 67
1 1 -912
3 3 -368
1 1 492
3 3 740
1 3 631
2 3 -9
2 3 -583
2 2 608
1 2 356
1 3 654
1 2 -57
2 2 918
2 3 298
1 3 319
2 3 -429
1 3 -118
1 1 -862
1 1 450
1 1 705
3 3 143
1 2 -254
1 3 -617
2 3 773
2 2 -338
2 2 -960
2 3 347
1 2 7
1 1 -283
1 2 -543
2 3 945
2 2 136
2 3 -204
1 3 446
1 1 828
1 3 315
1 1 -221
1 2 -320
1 3 783
2 2 -535
1 1 296
2 3 562
1 1 -789
2 2 -975
1 1 -591
3 3 188
1 2 -32
2 3 -966
2 3 -251
2 3 -551
1 2 532
2 2 469
3 3 132
1 2 908
2 2 429
1 2 -961
2 2 -228
1 2 972
1 1 727
3 3 -80
1 2 134
1 2 -686
1 3 -173
2 3 843
1 3 -549
1 1 113
1 1 -888
2 2 45
1 3 349
1 1 300
2 3 -592
1 1 260
2 3 -240
2 2 -474
1 2 120
2 2 813
2 2 896
2 3 940
1 3 -437
1 1 806
1 3 -412
3 3 197
1 2 -569
1 2 -360
1 3 337
1 2 -513
2 2 187
1 1 -861
1 1 314
2 3 13
2 3 -178
2 2 -690
1 3 659
1 3 -333
1 3 390
2 3 908
1 1 -4
1 1 449
1 3 -798
2 3 -319
1 3 -682
1 1 -98
2 2 940
1 3 475
2 3 -778
2 3 -526
1 1 -49
1 3 -708
2 2 -554
2 2 209
2 3 871
1 2 436
2 2 -132
2 3 -672
2 2 -375
1 3 723
1 2 -230
3 3 78
2 2 -801
2 3 -672
1 1 113
2 2 -709
2 3 -755
2 3 528
1 1 525
3 3 490
2 3 200
1 1 -345
1 3 -212
1 2 -317
1 1 -444
2 2 736
1 3 -782
1 1 -312
1 2 -476
1 2 672
1 3 635
2 3 -142
2 2 733
1 1 547
1 3 -744
1 2 461
2 2 16
2 2 935
3 3 622
1 2 182
2 2 -46
1 1 385
1 2 -412
2 3 -372
1 3 -97
2 2 891
1 2 483
1 2 -716
2 3 780
2 3 -485
1 2 -624
3 3 877
2 3 101
1 2 371
3 3 -123
1 2 40
2 2 -322
2 2 136
1 2 323
3 3 268
2 3 730
2 2 30
1 1 312
2 3 476
2 2 -592
2 2 422
1 2 -608
2 2 -425
1 2 -634
1 1 -852
2 2 -829
2 3 -278
3 3 -60
1 2 -883
1 2 -717
2 2 414
1 1 363
2 3 -533
3 3 533
2 3 -433
2 3 -19
1 1 -680
3 3 529
1 1 -436
1 2 630
2 2 -727
1 2 -616
1 3 546
3 3 -593
2 2 312
3 3 406
1 3 777
1 2 -345
1 1 731
1 1 -977
3 3 -177
2 2 734
1 3 356
1 3 602
1 2 704
2 2 -284
2 3 913
1 2 -454
2 2 -211
2 2 657
1 3 -868
2 2 -462
1 3 -308
3 3 766
1 3 626
2 3 535
1 1 -770
2 2 847
1 3 973
1 2 180
3 3 -244
2 2 866
2 2 -91
1 2 -362
3 3 -274
3 3 -597
2 3 793
1 3 704
1 1 -581
1 2 129
2 2 -610
1 3 550
1 2 -554
2 3 272
2 2 -728
3 3 -982
2 2 357
1 3 169
2 2 871
1 2 838
1 1 -675
2 3 -942
1 2 766
1 3 89
1 2 -710
1 2 664
1 3 439
1 3 -673
3 3 -240
3 3 499
1 1 -210
1 3 818
1 3 -593
2 2 578
1 2 -878
1 3 -114
2 2 -919
1 2 821
1 2 -445
2 2 -932
1 3 97
1 1 276
1 3 -876
1 3 838
1 1 184
3 3 818
2 2 959
1 1 -866
2 3 -657
1 1 -686
1 2 -721
1 1 243
2 2 486
1 2 -478
1 3 -556
3 3 249
1 3 -286
1 2 631
2 2 -488
2 2 -264
3 3 698
2 2 804
1 2 -815
1 2 973
3 3 836
1 2 -360
3 3 -836
2 3 866
2 2 -561
1 1 679
1 2 411
1 1 -17
1 3 668
1 3 451
2 2 167
1 2 347
3 3 -624
3 3 598
2 3 -951
3 3 -336
1 1 -326
1 2 853
2 2 199
1 3 558
1 1 23
2 3 -446
3 3 197
1 2 -613
1 1 -2
1 1 -186
2 2 -550
2 2 -18
1 1 -179
2 2 -189
2 2 -781
1 2 -626
1 2 -768
2 2 657
1 1 560
2 3 -346
2 2 429
2 3 791
1 1 -958
1 1 303
3 3 652
1 1 -76
3 3 -181
2 3 -136
2 2 -958
1 1 -777
2 2 153
2 3 668
1 3 682
1 1 -865
1 3 -254
1 1 -367
1 3 64
1 3 220
3 3 383
1 3 -419
2 2 -394
2 3 -462
3 3 -845
1 1 48
2 2 925
1 2 235
1 2 257
1 1 -359
3 3 -711
3 3 452
1 1 741
1 2 -237
3 3 414
1 1 -471
2 2 405
3 3 87
1 2 -602
1 1 49
1 2 -998
2 3 -139
2 2 686
3 3 -226
1 2 -931
2 2 354
2 2 -536
1 1 326
3 3 -657
1 1 888
3 3 -34
2 3 341
1 3 -145
3 3 467
2 2 -646
1 2 581
2 3 -271
3 3 -670
1 2 156
2 2 496
1 2 -494
2 3 -533
2 3 -767
2 3 491
1 1 -548
3 3 177
2 2 -101
2 3 -657
2 2 633
1 2 296
3 3 -196
3 3 -394
2 3 -199
3 3 138
1 2 -151
2 2 -687
1 1 373
1 3 -408
2 2 -739
1 2 -142
1 2 161
1 1 -891
3 3 595
3 3 -892
3 3 -271
1 2 497
1 1 85
2 2 -919
3 3 -1
1 2 -592
3 3 -360
3 3 695
3 3 -964
2 3 840
3 3 -137
1 3 903
1 3 -594
1 2 -724
1 3 368
2 3 -134
2 2 415
1 1 -407
1 1 215
1 3 -977
