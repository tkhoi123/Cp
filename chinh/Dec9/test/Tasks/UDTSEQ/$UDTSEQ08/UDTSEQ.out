-7271 -4609 6875 